src/components/scene/canvas.js

```diff
@@ -1,14 +1,14 @@
 var bind = require('../../utils/bind');
-var register = require('../../core/component').registerComponent;
-
-module.exports.Component = register('canvas', {
+var registerComponent = require('../../core/component').registerComponent;
 
+module.exports.Component = registerComponent('canvas', {
   init: function () {
+    var canvasEl;
     var sceneEl = this.el;
-    var canvasEl = document.createElement('canvas');
+
+    canvasEl = document.createElement('canvas');
     canvasEl.classList.add('a-canvas');
-    // Mark canvas as provided/injected by A-Frame.
-    canvasEl.dataset.aframeCanvas = true;
+    canvasEl.dataset.aframeCanvas = true;  // Mark canvas as provided/injected by A-Frame.
```

Comment:
I prefer to have comments just above the line they apply to. It's not always possible to put them in the same line. Due to the length of the code line or indentation.